web/boot.test.js

```diff
@@ -74,4 +74,12 @@ describe("textify!", () => {
       expect(await degausser(page, testCase.i)).toBe(testCase.o);
     });
   });
+
+  xit("xhtml", async() => {
```

Comment:
Note this test is skipped but I am fairly sure this module is no longer used (we use js module) and getting this test to work might take a considerable amount of time.